TabloidMVC/Controllers/TagController.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Microsoft.AspNetCore.Authorization;
```

Comment:
There are some unnecessary using statements at the top of this controller. The ones that are greyed out in Visual Studio can be removed.